src/medUtilities/medUtilities.cpp

```diff
@@ -142,7 +142,7 @@ void medUtilities::applyInverseOrientationMatrix(medAbstractView* view, double*
     std::copy(homogeneousVector, homogeneousVector + 3, outPoint);
 }
 
-void medUtilities::switchToVolumeRendering(medAbstractView *view)
+void medUtilities::switchTo3D(medAbstractView *view, QString mode3D)
```

Comment:
switchTo3D could take an enum defined in medUtilities (with values like VR, MR or whatever), and then in the code of this function you would have to have w switch on this enum to get the equivalent QString. I think it's better this way because the future programmer knows directly which values he/she can use just from reading the .h, instead of having to look at .cpp code. You don't HAVE to change it if you don't want to, I just think it would have been more ideal.